Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -220,7 +234,10 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
             break;
         }
 
-        if (!dontFireEvents)
+        if (this._subType)
+            value = `var(${value})`;
```

Comment:
if it's assumed that having a `_subType` means it will be a variable then we need to name `_subType` something better as `_subType` really doesnt indicate that at all.  how about `_variableType`?